CPT331.iOS/CPT331.iOS/EventManager.swift

```diff
@@ -62,14 +74,16 @@ class EventManager {
                     let name = event["name"].string
```

Comment:
I'm not sure how your able to use enums in Swift, but can you substitute these magic strings for the enum above?
Something like (not sure if its valid in swift):
```
let name = event[EventField.name].string
```